src/exceptionite/solutions.py

```diff
@@ -246,6 +264,61 @@ def regex(self):
         return r"unsupported operand type\(s\) for \+\: '(?P<type1>([\w\.]*))' and '(?P<type2>([\w\.]*))'"
 
 
+class MySQLConnectionRefused:
+    def title(self):
+        return "Could not connect to MySQL server"
+
+    def description(self):
+        return "Check that MySQL server is running and that MySQL configuration is correct (check that port, hostname, username and host are set correctly, and that environment variables are correctly defined)."
```

Comment:
I think should be check that port, hostname, username and **password** are set correctly